test-clients/src/main/java/com/hedera/services/bdd/spec/queries/token/HapiGetAccountNftInfos.java

```diff
@@ -0,0 +1,135 @@
+import com.hederahashgraph.api.proto.java.Transaction;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.Assert;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import static com.hedera.services.bdd.spec.queries.QueryUtils.answerCostHeader;
+import static com.hedera.services.bdd.spec.queries.QueryUtils.answerHeader;
+
+public class HapiGetAccountNftInfos extends HapiQueryOp<HapiGetAccountNftInfos> {
+    private static final Logger log = LogManager.getLogger(HapiGetAccountNftInfos.class);
+
+    String account;
+    long start;
+    long end;
+
+    public HapiGetAccountNftInfos(String account, long start, long end) {
+        this.account = account;
+        this.start = start;
+        this.end = end;
+    }
+
+    Optional<List<HapiTokenNftInfo>> expectedNfts = Optional.empty();
+
+    @Override
+    public HederaFunctionality type() {
+        return HederaFunctionality.TokenGetNftInfo;
+    }
+
+    @Override
+    protected HapiGetAccountNftInfos self() {
+        return this;
+    }
+
+    @Override
+    protected void assertExpectationsGiven(HapiApiSpec spec) throws Throwable {
+        var actualInfo = response.getTokenGetAccountNftInfos().getNftsList();
+        expectedNfts.ifPresent(nfts -> {
+            for (HapiTokenNftInfo nft : nfts) {
+                var expectedNftElement = TokenNftInfo.newBuilder();
+                var expectedNftId = NftID.newBuilder();
+
+                nft.expectedAccountID.ifPresent(e -> expectedNftElement.setAccountID(TxnUtils.asId(e, spec)));
+                nft.expectedTokenID.ifPresent(e -> {
+                    expectedNftId.setTokenID(TxnUtils.asTokenId(e, spec));
+                    var s = spec.registry().getCreationTimeSeconds(nft.expectedTokenID.get());
+                    var n = spec.registry().getCreationTimeNanos(nft.expectedTokenID.get());
+                    expectedNftElement.setCreationTime(Timestamp.newBuilder().setSeconds(s).setNanos(n).build());
+                });
+                nft.expectedSerialNum.ifPresent(expectedNftId::setSerialNumber);
+                expectedNftElement.setNftID(expectedNftId.build());
+                nft.expectedMetadata.ifPresent(expectedNftElement::setMetadata);
+
+                var completedNft = expectedNftElement.build();
+                Assert.assertTrue(actualInfo.contains(completedNft));
+            }
+            Assert.assertEquals(actualInfo.size(), nfts.size());
+        });
+    }
+
+    @Override
+    protected void submitWith(HapiApiSpec spec, Transaction payment) {
+        Query query = getAccountNftInfosQuery(spec, payment, false);
+        response = spec.clients().getTokenSvcStub(targetNodeFor(spec), useTls).getAccountNftInfos(query);
+        if (verboseLoggingOn) {
+            log.info("Info for '" + account + "': ");
+            response.getTokenGetAccountNftInfos().getNftsList().forEach(nft -> log.info(nft.toString()));
```

Comment:
1. It seems that the logging is broken -> it is separated into separate logs. 
2. We should indicate that this is the nft info for the account not simply "info for account"